tests/processor/test_orchestrator.py

```diff
@@ -51,19 +52,42 @@ def test_initializer(self):
 
 
     @patch('masu.external.report_downloader.ReportDownloader._set_downloader', return_value=FakeDownloader)
-    def test_download_curs(self, mock_downloader):
+    def test_prepare_curs(self, mock_downloader):
         """Test downloading cost usage reports."""
         orchestrator = Orchestrator()
 
-        reports = orchestrator.download_curs()
+        reports = orchestrator.prepare_curs()
 
         self.assertEqual(len(reports), 2)
 
     @patch('masu.external.report_downloader.ReportDownloader._set_downloader', return_value=FakeDownloader)
     @patch('masu.external.accounts_accessor.AccountsAccessor.get_accounts', return_value=[])
-    def test_download_curs_no_accounts(self, mock_downloader, mock_accounts_accessor):
+    def test_prepare_curs_no_accounts(self, mock_downloader, mock_accounts_accessor):
         """Test downloading cost usage reports."""
         orchestrator = Orchestrator()
-        reports = orchestrator.download_curs()
+        reports = orchestrator.prepare_curs()
 
         self.assertEqual(len(reports), 0)
+
+    @patch('masu.processor.tasks.process.process_report_file', return_value=None)
+    def test_process_curs(self, mock_task):
+        """Test downloading cost usage reports."""
+        requests = []
+        request1 = CURProcessRequest().report_path = '/test/path/file.csv'
+        requests.append(request1)
+        print((request1))
+        request2 = CURProcessRequest().report_path = '/test/path/file2.csv'
+        requests.append(request2)
+        print((request2))
+
+        orchestrator = Orchestrator()
+        orchestrator._processing_requests = requests
+        orchestrator.process_curs()
+
+    @patch('masu.processor.tasks.process.process_report_file', return_value=None)
+    @patch('masu.external.accounts_accessor.AccountsAccessor.get_accounts', return_value=[])
+    def test_process_curs_not_accounts(self, mock_task, mock_accounts_accessor):
+        """Test downloading cost usage reports with no pending requests."""
+        orchestrator = Orchestrator()
+
+        orchestrator.process_curs()
```

Comment:
seems to execute code and print stuff but not really assert anything.

Reply:
same as above